Test/WebTestCase.php

```diff
@@ -212,22 +213,6 @@ protected function getContainer()
     }
 
     /**
-     * @param string $omName
-     * @param string $registryName
-     *
-     * @return ObjectManager
-     */
-    protected function getObjectManager($omName = null, $registryName = 'doctrine')
```

Comment:
not sure why this method was ``protected`` to begin with .. and I doubt anyone has used this method as an extension point